packages/ragbits-document-search/src/ragbits/document_search/ingestion/providers/unstructured/default.py

```diff
@@ -70,13 +71,16 @@ def __init__(
                 variable will be used.
             api_server: The API server URL to use for the Unstructured API. If not specified, the
                 UNSTRUCTURED_SERVER_URL environment variable will be used.
+            use_api: whether to use unstructured API
```

Comment:
A more detailed documentation would be helpful - what is the alternative that will be used when `use_api` is set to `False`?